README.md

```diff
@@ -450,14 +451,14 @@ checkboxes against each row, on the left side of the table.
 Bulk select checkboxes are usually combined with bulk actions to perform
 actions on one or more rows at once.
 
-### Bulk actions
+### Actions
 
-Bulk actions, when combined with bulk select checkboxes allow you perform
+Actions, when combined with bulk select checkboxes allow you perform
 actions of multiple rows at once. When in use, a menu will be rendered
 in the top right of the table allowing your users to choose a bulk action
 that will be applied to the selected rows.
 
```

Comment:
```suggestion
Actions can also be used without bulk select checkboxes. This could allow for creation of action buttons that are not dependant on existing data, such as a 'Create User' button.
```